public/realms/mainnet-beta.json

```diff
@@ -3609,5 +3609,14 @@
     "twitter": "@EpicentralLabs",
     "discord": "https://discord.gg/5asAuY2sR8",
     "github": "https://github.com/EpicentralLabs"
-  }
+  },
+{ 
```

Comment:
can you try to ensure this is properly tabbed to the right.
Maybe the linter doesn't like it.